src/OpenEuropaLanguageSwitcherBuilder.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Drupal\oe_multilingual;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Security\TrustedCallbackInterface;
+
+/**
+ * Provides a trusted callback to alter the OpenEuropa language switcher.
+ *
+ * @see oe_multilingual_block_view_oe_theme_helper_page_header_alter()
+ */
+class OpenEuropaLanguageSwitcherBuilder implements TrustedCallbackInterface {
```

Comment:
No need to prefix the class name we have namespace, lets call it LanguageSwitcherBuilder